src/components/Licenses/Licenses.js

```diff
@@ -79,10 +92,46 @@ export default class Licenses extends React.Component {
         </AppIcon>
       );
     },
-    type: ({ type }) => type && type.label,
-    status: ({ status }) => status && status.label,
-    startDate: ({ startDate }) => (startDate ? <FormattedUTCDate value={startDate} /> : ''),
-    endDate: license => <LicenseEndDate license={license} />,
+    'type': ({ type }) => type && type.label,
```

Comment:
If you added the quotes around these keys because eslint complained, can you update this project to the latest version of `eslint-config-stripes` because we turned that rule off so that we _don't_ have to add these unnecessary quotes 😆